pkgs/development/python-modules/craft-parts/default.nix

```diff
@@ -94,6 +96,10 @@ buildPythonPackage rec {
       "tests/unit/packages/test_apt_cache.py"
       "tests/unit/packages/test_deb.py"
       "tests/unit/packages/test_chisel.py"
+      # Relies upon presence of java binaries
+      "tests/unit/plugins/test_ant_plugin.py"
+      "tests/unit/plugins/test_java_plugin.py"
+      "tests/unit/plugins/test_maven_plugin.py"
```

Comment:
I guess the missing dependencies can be added in `nativeCheckInputs` ?

Reply:
I understand, but I don't think it adds complexity. In any case, you decide.

Reply:
Fair enough, I've added the packages and reduced this to just a single test that I don't think it makes sense to patch in order for it to work here.